src/Tooltip/Tooltip.stories.tsx

```diff
@@ -314,6 +314,30 @@ const VisualTestTemplate: StoryFn<typeof Tooltip> = (args) => {
           Button
         </Button>
       </Tooltip>
+      <Tooltip
+        {...args}
+        maxwidth={327}
+        tooltipsize="Small"
+        placement="bottom-start"
+        title={(
+          <Box>
+            List
+            <ul style={{ margin: 0, paddingLeft: 20 }}>
+              <Typography variant="body2" sx={{ display: 'list-item' }}>Line 1</Typography>
+              <Typography variant="body2" sx={{ display: 'list-item' }}>Line 2</Typography>
+            </ul>
+          </Box>
+        )}
+      >
+        <Button sx={{
+          position: 'absolute',
+          left: '0',
+          marginLeft: '8px',
+        }}
+        >
+          Button
+        </Button>
+      </Tooltip>
```

Comment:
Hello @Omkumar Bhagat I see the slight issue now
It overlaps with one of the forced hovered on state already in the Visual Template Test (this one we use for Storyshots or Chromatic or whatever snapshot testing we have in place)

I would recommend just using the Interactive Template - you can change the title there if you want to a React.ReactNode. Thanks @ombhagat05 

See overlap - let's just avoid this altogether - please use the Interactive Example for your demo.
<img width="506" alt="Screenshot 2024-10-08 at 8 55 59 PM" src="https://github.com/user-attachments/assets/29c1b5f8-8000-49dc-91bf-8fd78ba8dbac">